src/components/NavBar/SignOutButton.tsx

```diff
@@ -15,9 +15,9 @@ const SignOutButton: React.FC = () => {
     };
 
     return (
-        <IconButton aria-label="Sign Out Button" onClick={handleSignOutClick}>
+        <Button color="secondary" aria-label="Sign Out Button" onClick={handleSignOutClick}>
```

Comment:
Shouldn't we keep this as an `IconButton`?

Reply:
You are right I changed it by accident